org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/internal/util/color/ColorUtils.java

```diff
@@ -43,7 +44,18 @@ public static Color getDecorationsColorFromCurrentTheme() {
      */
     public static ColorDescriptor getDecorationsColorDescriptorFromCurrentTheme() {
         ITheme theme = PlatformUI.getWorkbench().getThemeManager().getCurrentTheme();
+        System.out.println(theme.getColorRegistry().getKeySet());
```

Comment:
should this System.out be here?

Reply:
Of course, it's essential ;)
I've removed it.